src/module.ts

```diff
@@ -0,0 +1,34 @@
+import { defineNuxtModule } from '@nuxt/kit';
+
+import { FeedSource, FeedSourcesFactory } from './types';
+import { resolveModuleOptions } from './lib/resolveModuleOptions';
+import { generateFeedFile } from './lib/generateFeedFile';
+
+export * from './types';
+
+export interface ModuleOptions {
+  sources: FeedSourcesFactory | FeedSource[];
+}
+
+declare module '@nuxt/schema' {
+  interface NuxtConfig {
+    feed?: ModuleOptions;
+  }
+  interface NuxtOptions {
+    feed?: ModuleOptions;
+  }
+}
+
+export default defineNuxtModule<ModuleOptions>({
+  meta: {
+    name: '@nuxtjs/feed',
+    configKey: 'feed'
+  },
+  setup(moduleOptions, nuxt) {
```

Comment:
Now that SSG landed, the `feed` module could provide static feed generation directly too 😋